freestyle-logging/jvm/src/main/scala/loggingJVM.scala

```diff
@@ -50,4 +52,50 @@ object loggingJVM {
           ME.catchNonFatal(logger.warn(msg, cause))
       }
   }
+
+  implicit def freeStyleLoggingKleisli[M[_], C: Manifest](
+      implicit ME: MonadError[M, Throwable]): FSHandler[LoggingM.Op, Kleisli[M, Logger, ?]] =
+    λ[FunctionK[LoggingM.Op, Kleisli[M, Logger, ?]]](_ match {
+      case LoggingM.DebugOP(msg) =>
+        Kleisli { logger =>
```

Comment:
This can potentially be simplified to just have a single Kleisli wrapping all cases. Also now that I think about this close and being the intention to inject a Logger instance.. Would it make sense to have this just as an extra implicit constrain in the interpreter instead of interpreting to Kleisli?

In my opinion that would be a not so strong constrain since you would not need to provide the logging instance manually at runtime and it would be injected by the compiler at compile time. Also there is the work in #319  implemented in #352 that would need to be merged in this branch. @pepegar What are your thoughts?